src/table/table.scss

```diff
@@ -40,6 +40,10 @@
       cursor: pointer;
     }
 
+    &.iui-grabbing {
+      cursor: grabbing;
```

Comment:
You might also want to indicate somehow, that column is available for reorder.
`cursor: grab;`, again similar to slider

Reply:
I thought about that, but the entire column header is clickable and is using `cursor: pointer;`.

Reply:
Maybe a light border on hover?

Reply:
but wont it be weird, when cursor from pointer will change to grabbing ?

Reply:
At least on hover the cursor changes to the pointer hand, so its not a weird transition from `default` to hand.